java/ql/integration-tests/all-platforms/java/buildless-inherit-trust-store/test.py

```diff
@@ -0,0 +1,21 @@
+import sys
+
+from create_database_utils import *
+from buildless_test_utils import *
+from diagnostics_test_utils import *
+import subprocess
+import os.path
+
+repo_server_process = subprocess.Popen(["python3", "../server.py"], cwd = "repo")
```

Comment:
Below you go to some effort to make sure that `mypath` is relative to the integration test directory (rather than just using the working directory), but then here you just use `repo` which is presumably just relative to the current working directory?

Reply:
Either is fine in this instance. I absolute-ified the path passed to Java because I couldn't remember what the cwd would be for that process; cf here I know I'm the script and am spawning a simple process that doesn't change its cwd